src/flask/schemas/chef_schema.py

```diff
@@ -0,0 +1,24 @@
+# schemas fro serializing entities
+from datetime import datetime
+
+from marshmallow import Schema, fields, post_load
+
+from src.constants.audit import Status, InternalUsers
+from src.lib.entities.sqlalchemy_orm_mapping import Chef
+
+
+class ChefSchema(Schema):
+
+    user_id = fields.Int()
+    skill = fields.Int()
+
+    @post_load
+    def create_sqlalchemy_chef_entity(self, data, **kwargs):
+        chef = Chef()
+        chef.skill = data["skill"]
+        chef.user_id = data["user_id"]
+        chef.entity_status = Status.ACTIVE.value
+        chef.created_by = InternalUsers.SEEDER.value
```

Comment:
As mentioned above send this from the client

Reply:
done.